packages/experimental/Button/src/CompoundButton/CompoundButton.styling.ts

```diff
@@ -79,6 +41,10 @@ export const stylingSettings: UseStylingOptions<CompoundButtonProps, CompoundBut
     content: buildProps(
       (tokens: CompoundButtonTokens, theme: Theme) => ({
         style: {
+          marginLeft: 1,
+          marginRight: -1,
+          marginBottom: 1,
+          marginTop: -1,
```

Comment:
why the negative numbers here? Also Segoe UI workaround?

Reply:
Yeah, same Segoe thing here.

Reply:
I can potentially refactor this out

Reply:
Refactoring done.